apps/www/app/sink/new-york/page.tsx

```diff
@@ -58,7 +59,10 @@ export default function KitchenSinkPage() {
               <CardDemo className="w-full" />
             </ComponentWrapper>
             <ComponentWrapper>
-              <SliderDemo className="w-full" />
+              <SliderHoriztonalDemo className="w-full" />
+            </ComponentWrapper>
+            <ComponentWrapper className="flex h-[300px] w-full justify-center">
```

Comment:
if you remove this flex, the slider is growing to the sides, dont think it should be the regular behavior or mandatory to use flex

![image](https://github.com/shadcn-ui/ui/assets/13419087/e2ce0606-4480-48cf-b0bc-d018b5b667fc)